test/integration/manifests/load/privileged-daemonset-windows.yaml

```diff
@@ -19,6 +19,8 @@ spec:
     spec:
       nodeSelector:
         kubernetes.io/os: windows
+      tolerations:
+      - operator: Exists

```

Comment:
Is there a reason for adding a toleration for all taints ?
I see from the description that we want these pods to be scheduled always, no matter the taint on the node. Wondering what is the issue that blocked the pod to be scheduled. It should eventually be scheduled once the taint from the node is removed.